inception-ui-workload-management-and-monitoring/src/main/java/de/tudarmstadt/ukp/inception/app/ui/monitoring/page/MonitoringPage.java

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright 2017
+ * Ubiquitous Knowledge Processing (UKP) Lab
+ * Technische Universität Darmstadt
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+package de.tudarmstadt.ukp.inception.app.ui.monitoring.page;
```

Comment:
I'm getting checkstyle errors in this file because of wrong order of imports. When you build on the command line with Maven, you should be able to see the checkstyle errors as well. Otherwise, you can install a Checkstyle plugin into your IDE as described in the dev setup guide.